epoxy-adapter/proguard-rules.pro

```diff
@@ -15,3 +15,8 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
+
+-keep class * extends com.airbnb.epoxy.EpoxyController { *; }
+-keep class * extends com.airbnb.epoxy.ControllerHelper { *; }
+-keepclasseswithmembernames class * { @com.airbnb.epoxy.AutoModel <methods>; }
+-keepclasseswithmembernames class * { @com.airbnb.epoxy.AutoModel <fields>; }
```

Comment:
I don't think `keepclasseswithmembernames class * { @com.airbnb.epoxy.AutoModel <methods>;` is needed. The automodel annotation is only used on fields, not on methods.